### DISCUSSION TITLE
Anyone attending NAATW?

### DISCUSSION THREAD

**Discussion by dougie-wougie** [code4recovery/12-step-meeting-list#1190] [General]
Is anyone attending the 2023 NAATW Workshop? I signed up for the online, but I would like to know if anyone is also attending. I know Josh will be presenting. 

I hope to get insights to take my understanding to a new level. 

Doug

**Comment by brianw-area24** [comment]
Yes, Josh and Matt are presenting "Helping Online Intergroup Manage a Meeting Directory at Scale". Should be a really good presentation on the that project. I don't know if anyone else will be attending in person from Code for Recovery, but I think some may be online.

**Comment by lovethecodelabs** [comment]
I am attending NAATW, with topic AI in AA – Bringing New Depth to the Meeting List
(Matthew L., Long Beach)

**Reply by lovethecodelabs** [reply]
I am introducing ChatGPT with Alexa mostly for AA accessibility,  but there may be other purposes,  not currently tought about.

Presenting AI, json apis, python,  and node js.  Also a Alexa voice AI app is showcased

**Reply by dougie-wougie** [reply]
Thank you, Matt. That is one long list, awesome. Do you live in Long Beach, CA? I am in Santa Clarita. 

D

**Reply by lovethecodelabs** [reply]
Santa Clarita is not so far from where I grew up,  I live very close to long beach, now,  yes.   Long Beach is the main recovery intergroup office close to me.   Harbor Area.

**Reply by dougie-wougie** [reply]
Awesome. I started my aerospace career in Long Beach (at McDonnell Douglas, now Boeing) in the early '80s. I would love to meet in person soon. Have you ever attended the Pacific Group's big meeting on a Wednesday? They claim it is the largest regular meeting in the world. I grew up in La Crescenta. I have lived in about 12 LA/Orange County places and just moved back from Portland.

**Reply by lovethecodelabs** [reply]
I wrote a separate Alexa app in node js, years ago, to parse the tsml meeting info in json, and the share will cover the obstacles to passing information in bulk to AI, too.  

I'm finishing a second Alexa app, as a proof of concept,  interfacing with chatGPT directly

NodeJS will have to do the data analysis,  though,  as there are limits on how much info can be sent to chatGPT

Each request is made over an api call with json, and I like the practicality of python in Jupyter notebooks

My references
https://www.aa.org/meeting-guide-app
https://hacoaa.org/
https://hacoaa.org/wp-admin/admin-ajax.php?action=meetings
https://openai.com/
https://nodejs.org/en
https://developer.amazon.com/alexa/console/ask
https://www.python.org/
https://jupyter.org

**Reply by lovethecodelabs** [reply]
I've attended Bellflower big book on Mondays,  but never the Pacific Group.  That would be fun

**Reply by dougie-wougie** [reply]
I have never been a coder. I am (was) a software engineer. I considered myself an email engineer answering customer questions when I worked. I had a team that did the coding. I did large documentation for Northrop Grumman. We did over a million pages of what the Air Force called Technical Orders. My team and I converted documents from anything into SGML and then XML—boring stuff. 

These days, I do what I like to do. I spend a lot of time in Notion, my app of choice. 

I just finished a second BS, this one in Cyber Security during covid and then officially stopped working. I do four of the five AA sites here in the Santa Clarita area.

[Resume](https://folkus.notion.site/Douglas-Wade-Resume-cd8eae9469dd493ab8399077eab93d99?pvs=4)
[Folkus Site](https://folkus.cc/)

I have never used the links in Github before. So I hope I did it right.

**Reply by dougie-wougie** [reply]
My buddy from Portland (he moved to LA also) wanted to go to the Pacific group, we have gone a couple of times. He mentioned going to the Bellflower meeting. We will check it out.

**Reply by dougie-wougie** [reply]
Here are two of the sites I created. I am doing the Central Office (which got me working with the Meeting Guide) and our Convention, which is happening in October. 

[AA SCV Central Office](https://aascv.org/)
[AA SCV Convention](https://aascvconvention.org/) The woo-commerce shop is being released this week. They started it using EventBrite. Better late than never.

**Reply by lovethecodelabs** [reply]
Elegant!

**Reply by lovethecodelabs** [reply]
Where is pacific group on Wednesdays?

**Reply by lovethecodelabs** [reply]
I worked with Harbor Area Central Office in the past,  transitioning to wordpress and tsml, also

**Reply by dougie-wougie** [reply]
The Pacific group. 7:30 PM
Pacific Group meets 7:30 PM 
Veterans
Memorial Park
4117 Overland Avenue
Culver City, CA 90230

https://pg-aa.org/

**Reply by dougie-wougie** [reply]
They meet on Wednesday 7:30 PM

**Reply by dougie-wougie** [reply]
I use WordPress and the Divi framework. I am comfortable with it. Each framework has its strengths and weaknesses, but it is better to aim at mastering one over knowing a lot about everything.

D

**Reply by lovethecodelabs** [reply]
I used elementor in the past and was very satisfied with it,  kind of like divi

**Reply by lovethecodelabs** [reply]
Many obligations currently,  maybe pacific group in future...
